zipline/assets/continuous_futures.pyx

```diff
@@ -51,6 +51,7 @@ march_cycle_delivery_predicate = partial(delivery_predicate,
                                          set(['H', 'M', 'U', 'Z']))
 
 CHAIN_PREDICATES = {
+    'EL': march_cycle_delivery_predicate,
```

Comment:
'EL' is on March quarterly cycle: http://www.cmegroup.com/trading/interest-rates/stir/eurodollar_contract_specifications.html